devices/pine64-pinephonepro/README.adoc

```diff
@@ -0,0 +1,90 @@
+= Pine64 PinePhone Pro
+include::_support/common.inc[]
+
+== Device-specific notes
+
+=== Platform Firmware
+
+It is recommended to link:https://tow-boot.org/devices/pine64-pinephonePro.html[install Tow-Boot to SPI].
```

Comment:
```suggestion
The only *supported* platform firmware configuration is to link:https://tow-boot.org/devices/pine64-pinephonePro.html[install Tow-Boot to SPI].
```

Let's make it obvious it is not only recommended, but the only supported method.